### DISCUSSION TITLE
Now you can see your git history graphically on Github

### DISCUSSION THREAD

**Discussion by NirmalScaria** [community/community#131561] [Repositories]
Hello GitHub community! I'm excited to introduce a browser extension that enhances your GitHub experience by adding graphical Git history visualisation (a git graph) directly on repository pages.

<img width="1422" alt="Screenshot 2024-07-09 at 10 06 39 PM" src="https://github.com/community/community/assets/46727865/175c77ea-b8f0-49db-be88-65c899eaf2e7">

With this extension, you can:

- **Visualise Git History**: Explore your repository's commit history with an intuitive graphical representation.
- **Navigate Effortlessly**: Understand branch structures, merges, and commits at a glance.
- **Enhance Code Review**: Analyse changes and track development progress visually.

The extension is open source and has been published for chrome and firefox. I would love to hear what you think of it.

Source code : https://github.com/NirmalScaria/le-git-graph

Chrome based browsers : https://chromewebstore.google.com/detail/le-git-graph-commits-grap/joggkdfebigddmaagckekihhfncdobff

Mozilla Firefox : https://addons.mozilla.org/en-US/firefox/addon/le-git-graph-github-git-graph/

**Comment by control-owl** [comment]
Hi there. why dont you create a simple website showing this graph? I can not use this on a mobile phone.
